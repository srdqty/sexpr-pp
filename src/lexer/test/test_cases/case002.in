#| 
 alkdjflakjdlfkj 
 adskfjalkj3k4j3j 
    #| #|

|# #|

#||#|#
