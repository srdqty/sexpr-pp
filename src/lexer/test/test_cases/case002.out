stdin:4:5: end of file in #| comment
